brjs-core/src/main/java/org/bladerunnerjs/utility/deps/DependencyInfoFactory.java

```diff
@@ -113,6 +115,9 @@ private static void addAssetLocationDependencies(DependencyAdder dependencyAdder
 		DependencyInfo dependencyInfo, AssetLocation assetLocation) throws ModelOperationException {
 		for(LinkedAsset resourceAsset : assetLocation.linkedAssets()) {
 			dependencyInfo.resourceAssets.add(resourceAsset);
+			if (bundlableNode instanceof Workbench && assetLocation.assetContainer() instanceof Aspect) {
```

Comment:
Although I'm happy to incur some technical debt here, we should probably add a comment like:

```js
// TODO: the need for this code shows that our interfaces are incorrectly abstracted, which we should fix at some point
```

I assume too that we need this comment repeated in `BundleSetCreator`, or whatever the other class is that has a similar guard.